controllers/process.js

```diff
@@ -50,4 +47,57 @@ module.exports = ((req, res) => {
       ua.event('Entries', 'New entry error').send()
     }
   })
-})
+}
+
+function verifyCAPTCHA(req, res) {
+  return new Promise((resolve, reject) => {
+    staticman
```

Comment:
When chaining functions across multiple lines, I'd rather see the first one in the same line as the object being called on. So:

```js
staticman.getSiteConfig()
  .then()
  .catch()
```

Would you mind changing that?